examples/threathunter/threat_intelligence/threatintel.py

```diff
@@ -17,13 +17,19 @@ class ThreatIntel:
     def __init__(self):
         self.cb = CbThreatHunterAPI(timeout=200)
 
-    def push_to_cb(self, feed_id, results):
+    def verify_feed_exists(self, feed_id):
+        """Verify that a Feed exists"""
         try:
             feed = self.cb.select(Feed, feed_id)
+            return feed
         except ApiError as e:
-            log.error(f"couldn't find CbTH feed {feed_id}: {e}")
-            return
+            raise ApiError
+            return None
```

Comment:
You don't need to return because you raised the APIError and therefor you won't continue to the next line